docs/testing/stencil-testrunner/01-overview.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Stencil Test Runner Overview
+sidebar_label: Overview
+---
+
+# Overview
+
+Stencil has a built-in test runner that uses [Jest](https://jestjs.io/) and [Puppeteer](https://pptr.dev/) as its testing libraries, and allows developers to install both libraries using their preferred package manager.
+
+If you created a project using `npm init stencil`, these libraries were installed for you. Depending on when your project was created, you may or may not have the latest supported version installed.
+
+To view current version support for both Jest and Puppeteer, please see the  [Stencil support policy for testing libraries](../../reference/support-policy.md#testing-libraries).
```

Comment:
Wonder if we might want to add an `info` admonition in here calling out that you need to install `jest` and `puppeteer` if you don't have them - I imagine almost all new Stencil projects are going to be started with `npm init stencil` so users won't run into this, but still could be worth a call-out I think